src/main/kotlin/Calculator.kt

```diff
@@ -0,0 +1,73 @@
+class Calculator() {
+    fun calculate(expression: String?): Int {
+        nullOrEmptyCheck(expression)
+
+        val arr_experssion: Array<String> =
+            expression?.split(" ".toRegex())?.dropLastWhile { it.isEmpty() }?.toTypedArray() as Array<String>
+        val operando1 = arr_experssion[0].toInt()
+
+        val result = operatorCheck(operando1, arr_experssion)
```

Comment:
마이너하지만 early return 해서 불필요한 코드를 줄일 수 있을 거 같아요~
```suggestion
        return operatorCheck(operando1, arr_experssion)
```